examples/language_id_local_media_file.js

```diff
@@ -0,0 +1,50 @@
+const revai = require('revai-node-sdk');
+const token = require('./config/config.json').access_token;
+
+(async () => {
+    // Initialize your client with your Rev AI access token
+    var client = new revai.LanguageIdClient(token);
+
+    const jobOptions = {
+        metadata: 'node example language id local file submission',
+        notification_config: { url: 'https://jsonplaceholder.typicode.com/posts' },
+        delete_after_seconds: 30 * 24 * 60 * 60 // 30 days in seconds
+    };
+
+    // Media may be submitted from a local file
+    var job;
+    try {
+        job = await client.submitJobLocalFile('./resources/example.mp3',
+            jobOptions);
+    } catch (e) {
+        console.dir(e);
+    }
+
+    console.log('Language id job submitted.');
+    console.log(`Job Id: ${job.id}`);
+    console.log(`Status: ${job.status}`);
+    console.log(`Created On: ${job.created_on}`);
```

Comment:
I would move these logs into the try block

Reply:
shouldn't we only log this if the job successfully submitted?

Reply:
yea, so try will have the submit + all logs, catch will log the error, you had it already in the url sample file